sources/standard_pipeline.py

```diff
@@ -0,0 +1,119 @@
+import os
+from typing import Any
+
+import dlt
+from dlt.common.time import ensure_pendulum_datetime
+from pendulum.tz import UTC
+
+try:
+    from .standard.filesystem import filesystem_resource  # type: ignore
+except ImportError:
+    from standard.filesystem import filesystem_resource
+
+import pandas as pd
+from dlt.extract.source import TDataItem, TDataItems
+
+
+@dlt.transformer(name="filesystem")
+def copy_files(
+    items: TDataItems,
+    storage_path: str,
+) -> TDataItem:
+    """Reads files and copy them to local directory.
+
+    Args:
+        items (TDataItems): The list of files to copy.
+        storage_path (str, optional): The path to store the files.
+
+    Returns:
+        TDataItem: The list of files copied.
+    """
+    storage_path = os.path.abspath(storage_path)
+    os.makedirs(storage_path, exist_ok=True)
+    for file_obj in items:
+        file_dst = os.path.join(storage_path, file_obj["file_name"])
+        file_obj["path"] = file_dst
+        with open(file_dst, "wb") as f:
+            f.write(file_obj.read())
+        yield file_obj
+
+
+@dlt.transformer(  # type: ignore
+    table_name=lambda x: x["code"],
+    merge_key="date",
+    primary_key="date",
+    write_disposition="merge",
+)
+def extract_met_csv(
+    items: TDataItems,
+    incremental: dlt.sources.incremental[str] = dlt.sources.incremental(
+        "date",
+        primary_key="date",
+        initial_value="2023-01-01",
+        allow_external_schedulers=True,
+    ),
+) -> TDataItem:
+    """Reads files and copy them to local directory.
+
+    Args:
+        item (TDataItem): The list of files to copy.
+        incremental (dlt.sources.incremental[str], optional): The incremental source.
+
+    Returns:
+        TDataItem: The file content
+    """
+    for file_obj in items:
+        df = pd.read_csv(
```

Comment:
please demonstrate a stream writer where we use csv reader to read page by page with given size